no support for Unicode case-aware characters in Python re module  come on, reallyâ€½